e2e/scripts/cleanup-resources.sh

```diff
@@ -51,18 +48,8 @@ if ! type s3cmd > /dev/null 2>&1; then
   exit 1
 fi
 
-if [[ $# -gt 0 && $1 = "-h" ]]; then
-  usage
-  exit
-fi
-
-if [[ $# -ne 2 ]]; then
-  usage
-  exit 1
-fi
-
-readonly ID="$1"
-readonly DOMAIN="$2"
+readonly ID="${KOPS_CLUSTER_NAME}"
+readonly DOMAIN="${KOPS_CLUSTER_NAME}"
```

Comment:
I've been using `ccme2e.snodo.co` and everything seems to be getting cleaned up for me. 🤔 

It looks like it's because of:
```
$ doctl compute droplet list --format ID,Tags | grep "ccme2e.snodo.co"
152032167    KubernetesCluster:ccme2e-snodo-co
152032186    KubernetesCluster:ccme2e-snodo-co
152032187    KubernetesCluster:ccme2e-snodo-co
```

Reply:
@snormore hmm how does that `grep` work when there are dots in `ccme2e.snodo.co` but the tags use dashes like `ccme2e-snodo-co`? 🤔

Reply:
Apparently `.` is the single-character-wildcard in `grep` (TIL) 🤷‍♂